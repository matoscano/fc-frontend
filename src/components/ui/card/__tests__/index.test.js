import React from 'react';
import { render } from '@testing-library/react';
import Card from '../index';

it('renders a Card', () => {
  const { container } = render(
    <Card title="Title" width="200">
      Body
    </Card>,
  );
  expect(container).toMatchSnapshot();
});
