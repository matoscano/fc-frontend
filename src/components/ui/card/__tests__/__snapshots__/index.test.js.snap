// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Card 1`] = `
<div>
  <div
    class="sc-AxjAm jVZFDG"
  >
    <div
      class="sc-AxirZ eEcGbP"
    >
      <img
        alt="img movie"
        src="https://via.placeholder.com/150"
        title="Image movie"
      />
    </div>
    <div
      class="sc-AxiKw ceAJzB"
    >
      Body
    </div>
  </div>
</div>
`;
